README.md

```diff
@@ -1,34 +1,72 @@
-# Git Repository Template
+# Survey
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+[![codecov](https://codecov.io/gh/nimblehq/ic-flutter-avishek/branch/main/graph/badge.svg?token=ATUNXDX218)](https://codecov.io/gh/nimblehq/ic-flutter-avishek)
+
+All the templates that can be used to kick off a new Flutter application quickly.
 
 ## Usage
 
 Clone the repository
 
-`git clone git@github.com:nimblehq/git-template.git`
+`git clone git@github.com:nimblehq/flutter-templates.git`
```

Comment:
```suggestion
`git clone git@github.com:nimblehq/ic-flutter-avishek.git`
```

Reply:
Updated in https://github.com/nimblehq/ic-flutter-avishek/commit/087f4c2fa27bd775ebe3c3f2bbaaf109f6dbd957.